README.md

```diff
@@ -1,4 +1,8 @@
+
+# pandas-mentoring
+
 # Mentoring new pandas contributors (team 2019)
+
```

Comment:
Can you restore the original code here? I think all these blank lines and the old title are from fixing a conflict and not intended.